dev-lang/spidermonkey/spidermonkey-78.10.1.ebuild

```diff
@@ -230,6 +230,9 @@ src_prepare() {
 		python/mozbuild/mozbuild/configure/check_debug_ranges.py \
 		|| die "sed failed to set toolchain prefix"
 
+	# use prefix shell in wrapper linker scripts, bug #789660
+	hprefixify "${WORKDIR}"/firefox-${MY_PV}/build/cargo-{,host-}linker
```

Comment:
Could you please use path relative to `$S`?

Reply:
Sure, that would be like `${S}/../../build/...`, is that ok?